docs/standards/generic-standards/01-lsp2-json-schema.md

```diff
@@ -1,13 +1,19 @@
 ---
-sidebar_label: 'LSP2 - Storage Layout'
+sidebar_label: 'LSP2 - ERC725Y JSON Schema'
 sidebar_position: 3.3
 ---
 
-# The Storage Layout - LSP2 (JSON Schema)
+# LSP2 - ERC725Y JSON Schema
```

Comment:
do you think it makes sense to also put the name of the file like this?

01-lsp2-erc725y-json-schema.md

?

Reply:
@Hugoo will add that in a new PR